blazor/mention/code-snippet/display-template.razor

```diff
@@ -0,0 +1,52 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Data
+
+<SfMention TItem="EmployeeData" Query="@Query" PopupWidth="250px" suggestionCount=6>
```

Comment:
suggestionCount - WHy are use camel casing?

Have you ensured the all code snippet?

Reply:
Don't use the SuggestionCount property for all code snippets.